### ISSUE TITLE
unexpected token <

### ISSUE THREAD

**Issue by duncanhunter** [valor-software/ng2-charts#60]
I am able to use Chartjs making a custom directive with chartjs following this link http://stackoverflow.com/questions/34653045/third-party-js-in-angular2-typescript.

I am not able to get this library to work. Not sure what the error is as it is quite cryptic. I have tried referencing the d.ts file in the project with no changes to the error below.

Anyone had the same issue or who can point me in the right direction?

```
Uncaught SyntaxError: Unexpected token <
	Evaluating http://localhost:3000/ng2-charts/ng2-charts
	Error loading http://localhost:3000/app/main.js
```

**Comment by jkeczan** [comment]
I have the same issue

**Comment by valorkin** [comment]
Check readme.md in ng2-bootstrap project, you are missing system.js configuration part

**Comment by duncanhunter** [comment]
Hi

I am unsure what you mean as my system.js module loader is set up correctly? You have not specific set up instructions for system.js on this repo or ng2-bootstraps readme.md?

**Comment by valorkin** [comment]
https://github.com/valor-software/ng2-bootstrap#starter-packages

**Comment by jycouet** [comment]
Hi, 

I have the same issue.
I looked at the ng2-bootstraps and I've set systemJS like this:
`System.config({
        packages: {        
          app: {
            format: 'register',
            defaultExtension: 'js'
          }
        },
        map: {
          moment: 'node_modules/moment/moment.js',
        }
      });
      System.import('app/main')
            .then(null, console.error.bind(console));`

But it's not helping... Can you gide me?
What I'm doing wrong?

**Comment by valorkin** [comment]
Yeap, I need to add system.js bundling here to solve this once and for all :)

**Comment by jycouet** [comment]
Even with the code I pasted, it's not working!

**Comment by ccoat2** [comment]
I have the same problem... feel free if you have a solution to manifestate yourself (pleeasee!)

**Comment by TobyColeman** [comment]
+1

**Comment by dalecgu** [comment]
if you use SystemJs and npm,you should config like this.

```
      System.config({
        packages: {        
          app: {
            format: 'register',
            defaultExtension: 'js'
          },
          'ng2-charts': {
            defaultExtension: 'js'
          }
        },
        map: {
          'ng2-charts': 'node_modules/ng2-charts'
        }
      });
      System.import('app/main')
            .then(null, console.error.bind(console));
```

**Comment by valorkin** [comment]
should be fixed in #136 please check (release 1.0.2)
